### DISCUSSION TITLE
Allow removed org members to re-join the organization if SAML is still setup

### DISCUSSION THREAD

**Discussion by CallumYoung01** [community/community#23425] [New to GitHub]
<p>Hi Everyone,</p>
<p>Looking for some advice please,</p>
<p>We have noticed when removing a user from our organisations GitHub, the following is displayed:<br>
“Removed members will be able to rejoin the organization unless they are also deprovisioned from your SAML Identity Provider.”</p>
<p>Does anyone know how they can rejoin the org themselves? We have tested by removing a user from our org that was configured using SAML. When this removed user then goes to our org again, they only see public repos and have no option to rejoin.</p>
<p>Our hope with this is to remove users identified as inactive while leaving their SAML authentication in place. This way if/ when they need org access again, they could rejoin without the need for us to invite them manually.</p>
<p>Any thoughts and feedback would be most welcome.</p>
<p>Thank you.</p>

**Comment by djdefi** [comment, marked as answer]
<p>You could direct users to use the URL <code>https://github.com/orgs/ORGANIZATION/sso/sign_up</code> , replacing <em>ORGANIZATION</em> with the name of your organization. If they can authenticate against your configured SAML Identity Provider, then they should be able to join without an invite.</p>
<p>This is described within the following guide:</p>
<p><a href="https://docs.github.com/en/organizations/managing-saml-single-sign-on-for-your-organization/about-identity-and-access-management-with-saml-single-sign-on#adding-members-to-an-organization-using-saml-sso" class="onebox" target="_blank" rel="noopener nofollow ugc">https://docs.github.com/en/organizations/managing-saml-single-sign-on-for-your-organization/about-identity-and-access-management-with-saml-single-sign-on#adding-members-to-an-organization-using-saml-sso</a></p>

**Comment by CallumYoung01** [comment]
<p><a class="mention" href="/djdefi">@djdefi</a></p>
<p>Amazing, this does exactly what I was looking for. Thank you very much.</p>
